lib/circuitry/processors/threader.rb

```diff
@@ -0,0 +1,22 @@
+require 'circuitry/processor'
+
+module Circuitry
+  module Processors
+    module Threader
+      class << self
+        include Processor
+
+        def process(&block)
+          raise ArgumentError, 'no block given' unless block_given?
+          pool << Thread.new { safely_process(&block) }
```

Comment:
What are the dangers of unrestricted growth of this array? Would it make sense to implement something like threadpool from https://github.com/meh/ruby-thread to prevent an unhealthy proliferation of threads?